web/scripts/stripe/styling.css

```diff
@@ -0,0 +1,26 @@
+/**
```

Comment:
You can move and rename this file in web/styles/stripe.css

Reply:
+1

Reply:
Deleted the files. They were not required.